script.js

```diff
@@ -0,0 +1,21 @@
+// Declare Variables
+
+const menuButton = document.getElementById('menu-button');
+const menu = document.getElementById('menu-id');
+
+// Create event listener functions
+const showMenu = () => {
+  menu.classList.replace('hide', 'show');
+};
+
+const hideMenu = () => {
+  menu.classList.replace('show', 'hide');
```

Comment:
**Project Requirement**

> When the user clicks (or taps) the close (X) button, the mobile menu disappears.

- [ ] According to the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) When we click on the `X` button mobile nav should disappear.  I suggest you take a look at the `X` function again because the mobile nav doesn't disappear upon clicking `X`. (Note: It was working fine in the previous implementation take a look at the git history of your commits you may have accidentally deleted the code for functionality)